pkg/s3/signature4/v4-signature.go

```diff
@@ -349,7 +349,7 @@ func (s *Sign) DoesSignatureMatch(hashedPayload string) (bool, *probe.Error) {
 
 	// Verify if the access key id matches.
```

Comment:
Also the commit message and github PR message is completely different. May be a previous patch commit message.